physionet-django/user/templates/user/edit_cloud.html

```diff
@@ -1,10 +1,17 @@
 {% extends "user/settings.html" %}
 
-{% block title %} Edit Cloud information {% endblock %}
+{% block title %} User Cloud information {% endblock %}
 
 {% block main_content %}
-<h1 class="form-signin-heading">Manage cloud settings</h1>
+<h1 class="form-signin-heading">User Cloud information</h1>
 
+<p>Some of our databases are on Google Cloud Platform or Amazon Web Services.</p>
```

Comment:
I had a play around with the wording slightly to avoid hardcoding service names (so we can add integration without having to change it) and to provide a link to an example project.

Please could you use the version below if you're happy with it? Ideally `physionet` would be generated from a variable, rather than hardcoded.
```
PhysioNet is integrated with several cloud services. To access data via these services, you will need to:

- Provide the appropriate account identifier using the form below.
- Navigate to the "Files" section of the relevant project (for example, https://physionet.org/content/eicu-crd-demo/#files).
- Follow the instructions to request access. If instructions for cloud access are not shown, the project is not currently available on the cloud.
```